main/res/values/strings.xml

```diff
@@ -384,7 +385,7 @@
     <string name="caches_sort_favorites">Favorites</string>
     <string name="caches_sort_favorites_ratio">Favorites [%]</string>
     <string name="caches_sort_name">Name</string>
-    <string name="caches_sort_geocode">Geo Code</string>
+    <string name="caches_sort_geocode">Geocode</string>
```

Comment:
I believe "geo code" are two words in english?!

Reply:
I found only this as single word, see https://en.m.wikipedia.org/wiki/Geocode

Maybe we should use "cache code" to distinguish to other geographical information?

Reply:
OK, then lets just use "geocode" as one word as you suggest. I wasn't aware of that.